### ISSUE TITLE
discussion: how to flag properties to be used a HTTP headers

### ISSUE THREAD

**Issue by justin2004** [SPARQL-Anything/sparql.anything#66]
in order to talk to many REST APIs you need to add HTTP headers like the following example from amazon alexa:

```
Authorization: Bearer <<apiAccessToken>>
Accept-Language: <<locale, for example en-US>>
Accept: application/ld+json
```
https://developer.amazon.com/en-US/docs/alexa/custom-skills/linked-data-api-reference.html#header-of-the-request

in this [fork](https://github.com/justin2004/sparql.anything/tree/http-header-support) i've just been making all properties serve double duty as properties for sparql-anything and as http headers.

but what would be an acceptable way to specify http headers in a service clause?

maybe a property value that is a list?

> <x-sparql-anything:media-type=application/json,http-headers=(Accept=application/ld+json,Authorization=Bearer secret_token_here),https://some.com/api/resource>

**Comment by enridaga** [comment]
Dear Justin, this comes at the right time!
I am currently working on using Apache HTTP client and enable settings via properties.  I Will look into your fork to get a better idea of your use case. The update should cover HTTP methods, headers, and authentication, among other things. I am on leave now but should get this released at the beginning of August.

**Comment by enridaga** [comment]
Fixed with #67
